explorer/syntax/parser.ypp

```diff
@@ -551,21 +582,23 @@ additive_expression:
     {
       $$ = arena->New<OperatorExpression>(
           context.source_loc(), Operator::Add,
-          std::vector<Nonnull<Expression*>>({$1, $3}));
+          std::vector<Nonnull<Expression*>>(
+              {$[additive_lhs], $[additive_operand]}));
     }
 | additive_lhs MINUS additive_operand
     {
       $$ = arena->New<OperatorExpression>(
           context.source_loc(), Operator::Sub,
-          std::vector<Nonnull<Expression*>>({$1, $3}));
+          std::vector<Nonnull<Expression*>>(
+              {$[additive_lhs], $[additive_operand]}));
     }
 ;
 modulo_expression:
-  simple_binary_operand PERCENT simple_binary_operand
+  simple_binary_operand[lhs] PERCENT simple_binary_operand[rhs]
```

Comment:
Just a note, no change needed: for whatever reason, this already seems clear enough to me, and likewise for the other instances where you use `lhs` and `rhs` for a binary operator. I think it's because "lhs" and "rhs" have a very well-established meaning in this context, so it's easy to deduce what they refer to. So I wouldn't object to making them `lhs_operand` and `rhs_operand` or whatever, but I don't think it's necessary.

Reply:
Renamed per request.